mbedtls/src/hash/mod.rs

```diff
@@ -6,7 +6,7 @@
  * option. This file may not be copied, modified, or distributed except
  * according to those terms. */
 
-use crate::error::{Error, IntoResult, Result};
+use crate::error::{codes, IntoResult, Result};
```

Comment:
just a reminder:
need to ensure there is no usage of `Error` in this file

Reply:
done.